xml/schema/Core/MailSettings.xml

```diff
@@ -195,4 +195,88 @@
     <description>If this option is enabled, CiviCRM will not create new contacts when filing emails.</description>
     <add>5.31</add>
   </field>
+  <field>
+    <name>is_active</name>
+    <title>Enabled?</title>
+    <type>boolean</type>
+    <comment>Ignored for bounce processing, only for email-to-activity</comment>
+    <default>1</default>
+    <required>true</required>
+    <html>
+      <type>CheckBox</type>
+    </html>
+    <add>5.65</add>
```

Comment:
5.66 now.

Reply:
Done

Reply:
Will need to update the DAO at some point.